tests/phpunit/tests/block-bindings/render.php

```diff
@@ -114,6 +114,49 @@ public function test_passing_arguments_to_source() {
 		);
 	}
 
+	/**
+	 * Test passing `uses_context` as argument to the source.
+	 *
+	 * @ticket 60525
+	 *
+	 * @covers ::register_block_bindings_source
+	 */
+	public function test_passing_uses_context_to_source() {
+		$get_value_callback = function ( $source_args, $block_instance, $attribute_name ) {
+			$value = $block_instance->context['sourceContext'];
+			return "Value: $value";
+		};
+
+		register_block_bindings_source(
+			self::SOURCE_NAME,
+			array(
+				'label'              => self::SOURCE_LABEL,
+				'get_value_callback' => $get_value_callback,
+				'uses_context'       => array( 'sourceContext' ),
+			)
+		);
+
+		$block_content = <<<HTML
+		<!-- wp:paragraph {"metadata":{"bindings":{"content":{"source":"test/source", "args": {"key": "test"}}}}} -->
+		<p>This should not appear</p>
+		<!-- /wp:paragraph -->
+		HTML;
+		$parsed_blocks = parse_blocks( $block_content );
```

Comment:
Something is wrong here and needs to be investigated further:

<img width="616" alt="Screenshot 2024-02-15 at 11 52 25" src="https://github.com/WordPress/wordpress-develop/assets/699132/03ebc927-afac-44c2-b79f-7fb68a57f85f">

Reply:
It fails only on older versions of PHP, so my guess is that in line 143 you need to remove spaces:

```php
HTML;
```

Reply:
Just did it in [this commit](https://github.com/WordPress/wordpress-develop/pull/6079/commits/edaaffad0cceeef98ca9d982189057a25a364710). Let's see if that solves the problem. I guess it should because it is used in other tests that way.